components/link/create.js

```diff
@@ -58,7 +59,7 @@ export default function CreateLink(props) {
       r = r.replace(reg, words[idx])
     })
     return {
-      link: process.env.API_HOST + '/' + values.name + '/' + words.join('/'),
+      link: baseUrl + '/' + values.name + '/' + words.join('/'),
```

Comment:
probably should convert `baseUrl` to a function anyway so that we can apply logic to how the value is calculated.